bears/general/aspectsYEAHBear.py

```diff
@@ -0,0 +1,67 @@
+import re
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Diff import Diff
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.bearlib.aspects.Spelling import aspectsYEAH
+from coalib.bearlib.languages import Unknown
+
+
+class aspectsYEAHBear(LocalBear, aspects={
+        'detect': [aspectsYEAH],
+        'fix': [aspectsYEAH], }):
+    LANGUAGES = {'All'}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Spelling'}
+
+    def run(self, filename, file, aspects=[
```

Comment:
If you are willing to call it via aspects, the default value has to be `Root` perhaps instead of `aspectsYEAH`. Well we need to make it universal for all the bears therefore modify it in the run method of Bear class. Remember the discussion we had for the default value. I meant to place it here actually and by mistake I triggered a discussion in the instantiation of Bear class.